packages/keystone/plugins/addressService.js

```diff
@@ -124,6 +124,12 @@ const addressService = ({ fieldsHooks = {}, resolveAddressFields = ({ addressFie
                 addressFields['addressKey'] = get(result, 'addressKey')
                 addressFields['addressSources'] = get(result, 'addressSources', [])
             }
+        } else {
```

Comment:
Should validation error be here? Since you're trying to update an address info without changing address itself?

Delete seems strange here

Reply:
We have a bug here. You can change addressMeta